WordPress/src/androidTest/java/org/wordpress/android/e2e/ReaderTests.java

```diff
@@ -27,7 +26,7 @@ public void setUp() {
     String mCoachingPostTitle = "Let's check out the coaching team!";
     String mCompetitionPostTitle = "Let's focus on the competition.";
 
-    @Test
+//    @Test
```

Comment:
Question (❓): Why not ignoring this test instead of commenting it out?

PS.1: You actually chose to ignore some tests already (`sendButtonEnabledWhenTextIsEntered` + `loginWithEmailPassword`), while commenting out others (`navigateThroughPosts` + `likePost` + `allDayStatsLoad`). I would personally vote to ignore the tests instead of comment them out as they would be seen in the report as skipped (otherwise they would not be seen at all). Wdyt?
PS.2: The same comment applies to `ReaderTests.likePost()` and `StatsTests.allDayStatsLoad()`.

Reply:
Good question – running the test at all causes the test runner to crash, which fails the others, so unfortunately we can't simply ignore the results.